### PR TITLE
Frontend Tech Debt

### PR DESCRIPTION
**Pull request by conbrad**
- Rename `fbc` -> `fba` to unify fire behaviour advisory naming convention from backend
- Adds tests
- Test coverage is at 88.16% overall, 90.64% for `web/src/features/fbaCalculator`: https://codecov.io/gh/bcgov/wps/tree/0f46298688781d461f8fa39ccedd432867cdcc9c/web/src/features

### CODE REVIEW

File: web/.babelrc
Comment (conbrad) [inline web/.babelrc:2]: For `react-testing-library` to render components

File: web/src/features/fbaCalculator/components/GrassCureCell.tsx
Comment (andrea-williams) [inline web/src/features/fbaCalculator/components/GrassCureCell.tsx:59]: the `TextField` element in `WindSpeedCell` has an additional attribute `inputMode="numeric"`, which means that if users are using a mobile device to access the app, when they click into the cell, the number keyboard gets automatically rendered instead of the alphabetical one. Probably not very important because I don't think many (or maybe not any) of our users are accessing FireBat on mobile devices, but might be a nice-to-have just in case?

File: web/src/features/fbaCalculator/components/grassCureCell.test.tsx
Comment (andrea-williams) [inline web/src/features/fbaCalculator/components/grassCureCell.test.tsx:72]: FYI - if you're using the stepper buttons in the grass cure cell, the value maxes out at 100, but if you type in the value on a keyboard, you can still set values > 100 and the HFI, etc. calculations will increase accordingly (e.g., grass cure of 100 has HFI 4500; grass cure of 150 has HFI 5500).

Is this an opportunity for another unit test? A bug to be fixed in this ticket? Or in a separate ticket? Don't know. Just logging this while I still remember.

File: web/src/features/fbaCalculator/components/windSpeedCell.test.tsx
Comment (andrea-williams) [inline web/src/features/fbaCalculator/components/windSpeedCell.test.tsx:47]: fun insanity testing: if I use the keyboard to type in a stupidly large wind speed value, everything blows up and there's no error message (e.g., https://wps-prod.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=322&f=o1b&c=100&w=11111111)

Quick bug fix for this PR, or something to add to the backlog for later? I'll leave it up to you to decide.

File: web/src/features/fbaCalculator/components/grassCureCell.test.tsx
Comment (conbrad) [reply web/src/features/fbaCalculator/components/grassCureCell.test.tsx:72]: I think I can fix that issue in this PR, thanks!

Comment (andrea-williams) [reply web/src/features/fbaCalculator/components/grassCureCell.test.tsx:72]: I left a message in the General channel on Friday because I thought that capping the grass cure at a max of 100 might have been an assumption that we made, rather than knowing for sure, and I was feeling somewhat hesitant because it is theoretically possible for RH to go above 100. EK replied saying he wasn't sure whether grass cure can go above 100. Didn't get a reply from BB or DH - it miiiiight be worth confirming before you go to the trouble of actually doing the work.

Comment (conbrad) [reply web/src/features/fbaCalculator/components/grassCureCell.test.tsx:72]: Ok cool thanks for letting me know, I wouldn't have thought of that. I'll hold off on this change until it's confirmed then.

Comment (conbrad) [reply web/src/features/fbaCalculator/components/grassCureCell.test.tsx:84]: Limited grass cure input to 100 both via stepper and keyboard.

File: web/src/features/fbaCalculator/components/windSpeedCell.test.tsx
Comment (conbrad) [reply web/src/features/fbaCalculator/components/windSpeedCell.test.tsx:61]: Limited windspeed input to 100 both via stepper and keyboard.

File: web/src/features/fbaCalculator/components/GrassCureCell.tsx
Comment (conbrad) [reply web/src/features/fbaCalculator/components/GrassCureCell.tsx:67]: Added `inputMode="numeric"` property to `GrassCureCell`.

**Comment by andrea-williams** [comment]
If I type in a value > 100 into grass cure or wind speed, the calculations will still happen, I'll just see a red border around the cell. I'll defer to @tessmgood 's judgment on whether this amount of user feedback is sufficient.

**Review by andrea-williams — APPROVED** [review]
I'm happy with the refactors - letting Tess make the final call on error feedback to users re: >100 grass cure & wind speed values.

**Review by tessmgood — APPROVED** [review]

**Pull request merged**
